tests/macos.sh

```
@@ -242,7 +242,35 @@ else
+tokscale_err="$(mktemp -p "$TMPHOME")"
+if cz execute-template < "$TOKSCALE_SUBMIT_SOURCE" > "$rendered_tokscale_submit" 2>"$tokscale_err" \
+   && bash -n "$rendered_tokscale_submit" \
+   && grep -Fq 'default .chezmoi.hostname' "$TOKSCALE_SUBMIT_SOURCE" \
+   && grep -Fq 'export TZ="Asia/Seoul"' "$rendered_tokscale_submit" \
+   && grep -Fq 'bunx tokscale@latest submit </dev/null' "$rendered_tokscale_submit" \
+   && cz execute-template < "$TOKSCALE_PLIST_SOURCE" > "$rendered_tokscale_plist" 2>>"$tokscale_err" \
+   && plutil -lint "$rendered_tokscale_plist" >/dev/null 2>&1 \
+   && grep -Fq '<integer>14</integer>' "$rendered_tokscale_plist" \
+   && cz execute-template < "$TOKSCALE_LAUNCHD_SOURCE" > "$rendered_tokscale_launchd" 2>>"$tokscale_err" \
+   && bash -n "$rendered_tokscale_launchd" \
+   && grep -Fq 'launchctl bootstrap "gui/${uid}" "$plist"' "$rendered_tokscale_launchd" \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)